source/partials/release_notes/_release-18-1-0.md.erb

```diff
@@ -0,0 +1,98 @@
+* <%= link_to_commit 'addf343053658f79b91b9d7d39231cc94b434017', "Fix path of logback-include.xml", 4145%>
+* <%= link_to_commit 'd409d238f0c051e10857c57664ef62c4279c238b', 'Fix whitelist in configrepo hg material', 4191%>
+
+<h4>Other Improvements</h4>
+* <%= link_to_commit '23eb465f0b366378c526c70642c3d0748ce47e2e', 'Delete cruft from the DB for jobs that are completed and also for new jobs as they complete', 2875%>
+
+    From 18.1.0 release, jobs that are already completed (as per DB) and jobs that will complete from this release onwards, below data will be removed from DB:
+
+    * Environment variables that are not set using trigger-with-options
+
+    * Artifact plans that are of type \"file\" (Artifact plans of type \"unit\" are retained for shine)
+
+    * Artifact properties generators
+
+    * Agent resources associated with a job
+
+    * Elastic agent metatada
+
+
+<h4>Breaking Changes</h4>
+* As of release 17.5, authentication plugin endpoints have been deprecated. Support for authentication plugin endpoints has been removed in 18.1 release. The following plugins will not work unless been migrated to use the [Authorization Plugins](https://plugin-api.gocd.org/current/authorization/) endpoint:
+
+    <ul>
+      <li><p><%= link_to('Google OAuth Login', 'https://github.com/gocd-contrib/gocd-oauth-login') %>
+      (Alternative: <%= link_to('Google OAuth Authorization Plugin', 'https://github.com/gocd-contrib/google-oauth-authorization-plugin') %>)
+      </p>
+      </li>
+
+      <li><p><%= link_to('GitHub OAuth Login', 'https://github.com/gocd-contrib/gocd-oauth-login') %>
+      (Alternative: <%= link_to('GitHub OAuth Authorization Plugin', 'https://github.com/gocd-contrib/github-oauth-authorization-plugin') %>)
+      </p></li>
+
+      <li><p><%= link_to('GitLab(CE) OAuth Login', 'https://github.com/gocd-contrib/gocd-oauth-login') %>
+      (Alternative: <%= link_to('GitLab OAuth Authorization Plugin', 'https://github.com/gocd-contrib/gitlab-oauth-authorization-plugin') %>)
+      </p></li>
+
+      <li><p><%= link_to('TLS Client Certificates Login', 'https://github.com/cnorthwood/gocd-tls-auth') %>
+      (Use <%= link_to('latest version', 'https://github.com/cnorthwood/gocd-tls-auth/releases') %> of TLS Client Certificates Login instead.)
+      </p></li>
+
+      <li><p><%= link_to('Strong Auth Plugin', 'https://github.com/danielsomerfield/go-strong-auth-plugin#go-strong-auth-plugin') %>
+      (Use the bundled <%= link_to('gocd-filebased-authentication-plugin', 'https://github.com/gocd/gocd-filebased-authentication-plugin') %> instead.)
+      </p></li>
+      <li><p><%= link_to('Guest Login', 'https://github.com/gocd-contrib/gocd_auth_plugin_guest_user') %></p></li>
+    </ul>
+
+* <%= link_to_commit '23eb465f0b366378c526c70642c3d0748ce47e2e', 'Delete cruft from the DB for jobs that are completed and also for new jobs as they complete', 2875%>
```

Comment:
Remove this? You have already stated

Reply:
I think the line about `This improvement breaks jobs feed API. We will no longer render the environment variables, resources and build artifacts in the job xml api` needs to be added to the previous mention of this fix.